extensions/amp-dailymotion/0.1/amp-dailymotion.js

```diff
@@ -44,21 +100,54 @@ class AmpDailymotion extends AMP.BaseElement {
   }
 
   /** @override */
+  renderOutsideViewport() {
```

Comment:
it's fine to keep this as dailymotion embed seems pretty heavy but update the comments.